.circleci/config.yml

```diff
@@ -6,112 +6,44 @@ jobs:
       - image: circleci/node:8
     steps:
       - checkout
-    
-      - restore_cache:
-          name: Restore node modules cache
-          key: lock-v1-{{ checksum "package-lock.json" }}-{{ arch }}
-
-      - run: npm install 
-
-      - save_cache:
-          name: Save node modules cache
-          key: lock-v1-{{ checksum "package-lock.json" }}-{{ arch }}
-          paths:
-            - node_modules
-
+      - run: npm install
       - run:
-          name: jest tests
+          name: Test
           command: |
             mkdir -p test-results/jest
-            npm run test-js
+            npm run test
           environment:
             JEST_JUNIT_OUTPUT: test-results/jest/junit.xml
-
       - persist_to_workspace:
           root: ~/react-native-radar
           paths:
             - node_modules
-
       - store_test_results:
           path: test-results
-
       - store_artifacts:
           path: test-results
-
   android:
-    working_directory: ~/react-native-radar
+    working_directory: ~/react-native-radar/example
     docker:
       - image: circleci/android:api-28-node
     steps:
       - checkout:
-
-      - attach_workspace:
-          at: ~/react-native-radar
-      
-      - run: npm run test-android
-
-      - run:
-          name: Save test results
-          command: |
-            mkdir -p test-results/android/
-            find ./android -type f -regex ".*/build/test-results/.*xml" -exec cp {} ./test-results/android/ \;
-          when: always
-
-      - store_test_results:
-          path: test-results
-
-      - store_artifacts:
-          path: test-results
-
+          path: ~/react-native-radar
+      - run: yarn install
+      - run: sudo npm install -g react-native
```

Comment:
seems odd at first glance to do a `yarn install` and then a `npm install`, are both package managers used here?

Reply:
`yarn` used to install dependencies, `npm` used to install `react-native` command line tools for builds. I'll see if I can just install it with `yarn` for consistency